### DISCUSSION TITLE
zeromq 不是 GPL 3

### DISCUSSION THREAD

**Discussion by DanielKui** [xmake-io/xmake#4037] [General]
![image](https://github.com/xmake-io/xmake/assets/578118/cae590dd-ce4b-4ac5-a8fc-3033f58257d8)

![image](https://github.com/xmake-io/xmake/assets/578118/760b4b55-4e39-496b-961e-715c85f767b1)

set_license("MPL-2.0") 才对吧？

现在add_requires("zeromq")后，各种GPL 3 warning

**Comment by waruqi** [comment]
来个 pr 改下
